scio-bigquery/src/it/scala/com/spotify/scio/bigquery/types/StorageIT.scala

```diff
@@ -316,6 +316,7 @@ object StorageIT {
   @BigQueryType.fromStorage("data-integration-test:partition_a.table_%s", List("$LATEST"))
   class StorageLatest
 
+  // Require setting default project to allow this work locally. Use command `gcloud config set project data-integration-test`.
```

Comment:
```suggestion
  project data-integration-test`.
```

I think this comment is better added in the docs. I'll purpose smth soon.